backend/integration.go

```diff
@@ -2756,3 +2759,67 @@ func testStaffInvasion(s *serverUnderTest) {
 			`{"session_id":"%s","id":"*","name":"","server_id":"*","server_era":"*","is_staff":true,"is_manager":true}`, c2.sessionID)
 	})
 }
+
+func testNotifyUser(s *serverUnderTest) {
+	Convey("Successful login disconnects all sessions associated with user", func() {
+		b := s.backend
+		ctx := scope.New()
+		kms := s.app.kms
+
+		// Create manager account and room.
+		nonce := fmt.Sprintf("notify-%s", time.Now())
+		cammie, _, err := s.Account(ctx, kms, "email", "cammie"+nonce, "cammiepass")
+		So(err, ShouldBeNil)
+
+		_, err = b.CreateRoom(ctx, kms, false, "notify1", cammie)
```

Comment:
You don't need to create rooms in advance in these tests, unless having a user be a manager is relevant to the test.